shared/profile/render.desktop.js

```diff
@@ -113,11 +113,11 @@ class Render extends Component<void, Props, State> {
             borderBottom: `1px solid ${globalColors.black_05}`,
           }}
         >
-          <PlatformIcon platform={proof.type} overlay='iconfont-proof-good' overlayColor={globalColors.blue} size={48} />
+          <PlatformIcon platform={proof.type} overlay='icon-proof-success' overlayColor={globalColors.blue} size={48} />
           <Text type='Body' style={{textAlign: 'center', color: globalColors.black_40}}>Posted on<br />{moment(proof.mTime).format('ddd MMM D, YYYY')}</Text>
```

Comment:
We should standardize this date format somewhere. Maybe in `styles/style-guide`?